mrmustard/utils/settings.py

```diff
@@ -14,6 +14,8 @@
 
 """A module containing global settings."""
 
+from __future__ import annotations
```

Comment:
Once @kaspernielsen96 verifies that this works we should probably convert all of our `Optional` type hints to `|` and include this import. I can open

Reply:
It works with these updates on 3.9.